guides/deployment/heroku.md

```diff
@@ -230,11 +208,7 @@ end
 
 This ensures that any idle connections are closed by Phoenix before they reach Heroku's 55-second timeout window.
 
-Lastly, we'll need to create a [Procfile](https://devcenter.heroku.com/articles/procfile) (a text file called "Procfile" in the root of our project’s folder) with the following line:
-
-```
-web: MIX_ENV=prod elixir --sname server -S mix phx.server
-```
+As is common with Heroku's buildpacks, you can create your own "Procfile", but for our case we will use the default one provided by the [heroku-buildpack-phoenix-static buildpack](https://github.com/gjaldon/heroku-buildpack-phoenix-static).
```

Comment:
We are adding two build packs and both have a default procfile. How does Heroku chooses which one to keep?

Reply:
I added [hopefully] a clarifying sentence for this scenario. See the commit below.